frontend/src/pages/gov/chain/ProposalInfo.js

```diff
@@ -210,7 +218,8 @@ export default function ProposalInfo() {
           aminoConfig: chainInfo.aminoConfig,
           prefix: chainInfo.config.bech32Config.bech32PrefixAccAddr,
           feeAmount:
-            chainInfo.config?.feeCurrencies?.[0]?.gasPriceStep.average * 10 ** currency.coinDecimals,
+            chainInfo.config?.feeCurrencies?.[0]?.gasPriceStep.average *
```

Comment:
why shouldn't we write one utility function  **getAverageFee** to get the average fee?